sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -2593,19 +2597,27 @@ SIREPO.app.directive('emailLogin', function(requestSender, errorService) {
         restrict: 'A',
         scope: {},
         template: [
-            '<div class="row text-center">',
-              '<p>Enter your email address and we\'ll send an authorization link to your inbox.</p>',
+            '<div data-ng-show="isJupyterhub" class="alert alert-info col-sm-offset-2 col-sm-10" role="alert">',
+            'We’re improving your Jupyter experience by making both Jupyter and Sirepo accessible via a single email login. Simply follow the directions below to complete this process.',
```

Comment:
apostrophe is confusing vs single quote. rather use backslash single quote for `We\'re`.

Reply:
8f6dbc4f